app-text/cutemarked/cutemarked-0.11.3.recipe

```diff
@@ -0,0 +1,70 @@
+SUMMARY="Qt Markdown editor"
+DESCRIPTION="A Qt-based, free and open source Markdown editor with live HTML preview, math expressions, code and markdown syntax highlighting."
+HOMEPAGE="https://cloose.github.io/CuteMarkEd"
+COPYRIGHT="2013 Christian Loose"
+LICENSE="GNU GPL v2
+		 MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/cloose/CuteMarkEd/archive/v$portVersion.zip"
+CHECKSUM_SHA256="a8f1c8f22ce52ff8a878607112a44ca3d45015120d07b301f78fc9631b894775"
+SOURCE_DIR="CuteMarkEd-$portVersion"
+
+ARCHITECTURES="!x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+		cutemarked$secondaryArchSuffix = $portVersion
+		app:CuteMarkEd$secondaryArchSuffix = $portVersion
+		"
+		
+REQUIRES="
+		haiku$secondaryArchSuffix
+		hunspell$secondaryArchSuffix
```

Comment:
Does this requires `hunspell` as a whole, or only a part of it?

Reply:
I did like that  because HaikuPorter can't resolve devel:libhunspell, which is contained in that package.

Reply:
https://github.com/haikuports/haikuports/blob/master/app-text/hunspell/hunspell-1.6.2.recipe#L48

Reply:
Ok, then how can I get a version of hunspell in recipe?

Reply:
You can't unfortunately. As of now this would requires `devel:libhunspell_1.6$secondaryArchSuffix` and `lib:libhunspell_1.6$secondaryArchSuffix`

Reply:
ok